src/org/usfirst/frc/team2399/robot/commands/JoyShoot.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc.team2399.robot.commands;
+
+import org.usfirst.frc.team2399.robot.OI;
+import org.usfirst.frc.team2399.robot.Robot;
+import org.usfirst.frc.team2399.robot.subsystems.Shooter;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class JoyShoot extends Command {
+	
+	double speed;
+	private Shooter shooter = Robot.shooter;
+
+    public JoyShoot(double speed) {
+    	this.speed = speed;
+    	requires(shooter);
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    }
+
+    //TODO: Incorporate throttle (see last year's code)
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    	
+    }
+
+    //When the command is run, the shooter is set to this speed
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+    	shooter.setShooterSpeed(speed * OI.getShooterThrottle());
+    }
+
+    // Make this return true when this Command no longer needs to run execute()
+    protected boolean isFinished() {
+        return false;
```

Comment:
Nothing makes this stop currently. Is there a reason to stop joystick control of the shooter once teleop has begun?